README.md

```diff
@@ -130,19 +156,242 @@ Usage: txid2txref [options] <txid>
+ --rpcport [port]           RPC port (default: try both 8332 and 18332) 
+ --config [config_path]     Full pathname to bitcoin.conf (default: <homedir>/.bitcoin/bitcoin.conf) 
+ --utxoIndex [index]        Index # of which UTXO to use from the input transaction (default: 0) 
+
+<inputXXX>      input: (bitcoin address, txid, txref, or txref-ext) needs at least slightly more unspent BTCs than your offered fee
+<outputAddress> output bitcoin address: will receive transaction change and be the basis for your DID
+<network>       the bitcoin network you want to use, 'main' or 'test'
+<WIF>           WIF representation of your private key
+<fee>           fee you are willing to pay (suggestion: >0.001 BTC)
+<ddoRef>        reference to a DDO you want as part of your DID (optional)
+```
+
+Many of the runtime options for createBtcrDid are for connecting to
+bitcoind over RPC. There are four `--rpc*` options that you can use for
+connecting to a local or remote bitcoind:
+
+```
+$ ./src/createBtcrDid --rpcuser bitcoinrpc --rpcpassword super-secret-passwd --rpchost 127.0.0.1 --rpcport 18332 ...
+```
+
+If you have bitcoind running locally, you probably have a bitcoin.conf
+file with all the RPC info in there. If so, you can use it instead of
+setting all of the `--rpc*` options:
+
+```
+$ # by default it will look in <homedir>/.bitcoin/bitcoin.conf
+$ ./src/createBtcrDid ...
+{...}
+
+$ # or you can use a different location
+$ ./src/createBtcrDid --config /tmp/bitcoin.conf ...
+{...}
+```
+
+createBtcrDid has many required positional parameters:
+
+#### inputXXX      
+This is the input transaction--where you need to have at least slightly
+more unspent BTCs than your offered fee (see below). You can refer to
+this transaction in many ways: a bitcoin address (plus utxoIndex, given
+with `--utxoIndex` parameter); a txid (plus utxoIndex); a txref (plus
+utxoIndex), or a txref-ext.
+
+#### outputAddress 
+This is the output bitcoin address. It will receive transaction change and be the basis for your DID.
+
+#### network
+the bitcoin network you want to use, 'main' or 'test'
+
+#### WIF
+WIF representation of your private key
+
+#### fee
+fee you are willing to pay (suggestion: >0.001 BTC)
+
+#### ddoRef
+reference to a DDO you want as part of your DID. Usually a URL to a
+JSON-LD document. (optional)
+
+## Example of how to use createBtcrDid
+
+Some prerequisites:
+- you need a running instance of bitcoind, with the option "txindex=1"
+enabled so you have full transaction history. 
+- you need to have some BTC, either on the main or
+test blockchain (whatever your bitcoind is running)
+- you need the txid of a transaction with your BTC, and the index of the
+UTXO you want to use (if it is other than 0)
+- if you want your DID to have a reference to an external document with
+more info (a DDO) then make sure you have a URL for where it can be
+retrieved from.
+
+Make sure you know the txid and how much you have to spend. You can use
```

Comment:
maybe "how much you can spend from it", to differentiate from fee requirements.

Reply:
Yes, thanks.